cmd/podman/varlink/io.podman.varlink

```diff
@@ -658,8 +658,13 @@ method PauseContainer(name: string) -> (container: string)
 # See also [PauseContainer](#PauseContainer).
 method UnpauseContainer(name: string) -> (container: string)
 
+# Attach takes the name or ID of a container and sets up a the ability to remotely attach to its console.
 method Attach(name: string) -> ()
 
+# Attach takes the name or ID of a container and sets up a the ability to remotely attach to its console
+# after the container is started.
+method StartAndAttach(name: string) -> ()
+
 method AttachControl(name: string) -> ()
```

Comment:
What does this do?

Reply:
These are older API endpoints that are used to discover the paths to the sockets